app/scaffold-config/tabConfig.js

```diff
@@ -0,0 +1,56 @@
+define(['config/baseConfig'], function(BaseConfig) {
```

Comment:
Do you think it would be worthwhile to have some re-usability between tabConfig and menuConfig? Or are those totally different concepts now?